src/components/DuosHeader.js

```diff
@@ -149,7 +177,21 @@ class DuosHeader extends Component {
 
                   li({}, [
                     h(Link, { id: 'link_help', to: helpLink }, ['Request Help'])
-                  ])
+                  ]),
+
+                  button({
+                    id: "btn_applyAcces",
+                    className: "contact-us-button",
+                    onClick: this.SupportRequestModal,
+                    "data-tip": "Need help? Contact us for some assistance", "data-for": "tip_requestAccess"
+                  }, ["Contact Us"]),
+                  SupportRequestModal({
+                    showModal: this.state.showSupportRequestModal,
+                    onOKRequest: this.okSupportRequestModal,
+                    onCloseRequest: this.closeSupportRequestModal,
+                    onAfterOpen: this.afterSupportRequestModalOpen,
+                    url: this.props.location.pathname
+                  }),
```

Comment:
To avoid duplication, I think you can make this button a const at the top of the render method then just call the const here and below.